### PR TITLE
feat(ctp): fix erc721 bridge related contracts

### PR DESCRIPTION
**Pull request by sam-goldman**
Contains all of the fixes for the ERC721 bridge from the OpenZeppelin audit. Bumps the semver version to 1.0.0.

### CODE REVIEW

File: packages/contracts-periphery/contracts/L1/L1ERC721Bridge.sol [hunk 1]
```diff
@@ -75,87 +26,20 @@ contract L1ERC721Bridge is Semver, CrossDomainEnabled, OwnableUpgradeable {
      * @param _messenger   Address of the CrossDomainMessenger on this network.
      * @param _otherBridge Address of the ERC721 bridge on the other network.
      */
-    constructor(address _messenger, address _otherBridge)
-        Semver(0, 0, 1)
-        CrossDomainEnabled(address(0))
-    {
+    constructor(address _messenger, address _otherBridge) Semver(0, 0, 1) {
```
Comment (maurelian) [inline packages/contracts-periphery/contracts/L1/L1ERC721Bridge.sol:29]: Can you bump this to 1.0.0?

**Comment by maurelian** [comment]
Can you rebase this on upstream/develop and open a PR there please?

**Comment by sam-goldman** [comment]
> Can you rebase this on upstream/develop and open a PR there please?

Do you mean the develop branch in optimism-audit-fixes or in the regular monorepo?

**Comment by sam-goldman** [comment]
Closed by [this PR](https://github.com/ethereum-optimism/optimism/pull/3376) in Optimism's monorepo.

**Pull request closed without merging**